nio_cli/cli.py

```diff
@@ -2,21 +2,23 @@
 nio
 
 Usage:
-  nio new <project-name> [(--template <template> | -t <template>)] [(--pubkeeper-hostname=HOSTAME --pubkeeper-token=TOKEN)]
+  nio new <project-name> [(--template <template> | -t <template>)] [(--pubkeeper-hostname=HOSTAME --pubkeeper-token=TOKEN)] [--username=] [--password=]
   nio newblock <block-name>
   nio [options] add <block-repo>... [(--upgrade | -u)]
-  nio [options] (list | ls) services
-  nio [options] (list | ls) blocks
-  nio [options] (command | co) <command-name> [<service-name>] [<block-name>] [--args=<args>]
-  nio [options] (config | cfg) [project]
-  nio [options] (config | cfg) services <service-name>
-  nio [options] (config | cfg) blocks <block-name>
-  nio [options] clone services <service-name> <new-name>
+  nio [options] (list | ls) services [--username=] [--password=]
```

Comment:
If these are included in the `[options]` section then I don't think we need to document for every single CLI command. I would remove from the docstring and just consider them part of options